internal/testrunner/runners/system/runner.go

```diff
@@ -51,17 +51,6 @@ type runner struct {
 	wipeDataStreamHandler   func() error
 }
 
-type stackSettings struct {
```

Comment:
Is the `stackSettings` (old and new) structure used anywhere? It looks like a leftover after refactoring. All stack settings like host, login, password don't need to be exposed beyond ES/Kibana clients.

Reply:
Good catch. I've removed all stack settings code in cef5f22e598073727598a02d7324ffe5484f4f1c. So now the ES/Kibana clients directly call `os.Getenv` in their constructors to get the stack settings.